packages/near-membrane-dom/src/window.ts

```diff
@@ -114,7 +114,7 @@ function filterWindowDescriptors(
     // iframe, we can let the sandbox to use the iframe's api directly,
     // instead of remapping it to the blue realm.
     // TODO [issue #67]: review this list
-    delete to.crypto;
```

Comment:
The reason we were removing `crypto` global from the remapping list was precisely the inverse of the issue `W-10350344`, basically, because we were not remapping the typed arrays, trying to use the crypto from blue will complaint about the arguments not being typed arrays, so we decided at the time to preserve the original crypto per realm (that was going to be problematic when we start using the ShadowRealm because that will probably not have crypto, or might not have it). Anyhow, this PR makes all typed arrays blue objects, meaning crypto has to be blue as well in order to operate well with those blue typed arrays.

Reply:
it is a comment for now, eventually we need to clean this PR anyways.

Reply:
AH! This is what caused me to back out the previous change.

Reply:
Let's remove these two lines all together, including the TODO since it is not longer valid.